inert/dynamic-inert-on-focused-element.tentative.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Dynamic inertness on focused element</title>
+<link rel="author" title="Oriol Brufau" href="mailto:obrufau@igalia.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#focus-fixup-rule">
+<meta name="assert" content="If a focused element becomes expressly inert, it stops being focused.">
+<div id="log"></div>
+
+<input class="has-focus becomes-inert">
+
+<div class="becomes-inert">
+  <input class="has-focus">
+</div>
+
+<button class="has-focus becomes-inert">foo</button>
+
+<div class="has-focus becomes-inert" tabindex="-1"></div>
+
+<div class="becomes-inert">
+  <div class="has-focus" tabindex="-1">bar</div>
+</div>
+
+<div class="becomes-inert">
+  <p>
+    <span class="has-focus" tabindex="-1">baz</span>
+  </p>
+</div>
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+function nextRepaint() {
+  return new Promise(resolve => {
+    requestAnimationFrame(() => {
+      requestAnimationFrame(resolve);
+    });
+  });
+}
+
+promise_setup(nextRepaint);
+
+for (const element of document.querySelectorAll(".has-focus")) {
+  const becomesInert = element.closest(".becomes-inert");
+  promise_test(async function() {
+    element.focus();
+    assert_equals(document.activeElement, element, "The element is focused");
+
+    becomesInert.inert = true;
+    await nextRepaint();
```

Comment:
Should it really wait for the next repaint?

Reply:
Possibly not, but Blink also needs to wait for a repaint with `becomesInert.style.visibility = "hidden"`, so it seems a problem of the feature itself, not specific to inert. So while this makes the test weaker, it's not wrong, and the stricter check could be tested in a non-inert test.

Reply:
Ok, please add a note to the relevant chromium bug (if any) to remove this workaround from the test.